### PR TITLE
update Acorn, add Snapmail (because Windows support)

### PR DESCRIPTION
**Pull request by Connoropolous**
see it: https://deploy-preview-301--fervent-tesla-2a3da2.netlify.app/bluesky-try/

### CODE REVIEW

File: src/bluesky-try.md [hunk 1]
```diff
@@ -21,5 +21,6 @@ If you want to get started writing an application with Holochain, go to the [hom
...
-* Download [Acorn](https://github.com/h-be/acorn), a collaborative planning app. If you want to join a public workspace, try the project invitation secret `residual improvise landfall bogus degraded`. (Warning: it' s an unmoderated space, so don't be surprised if you see weird stuff there.)
-* If you don't mind the command line, download the latest [Holochain Launcher and Elemental Chess](https://github.com/holochain/launcher/releases/) and play against a friend. (Launcher is still pre-alpha, so you may encounter some bugs.)
\ No newline at end of file
+* If you are on Linux or Mac, download [Acorn](https://github.com/h-be/acorn/releases/tag/v0.5.1-alpha), a collaborative planning app. If you want to join a public workspace, try the project invitation secret `residual improvise landfall bogus degraded`. (Warning: it's an unmoderated space, so don't be surprised if you see weird stuff there.)
+* If you are on Linux or Mac, download the latest [Holochain Launcher and Elemental Chess](https://github.com/holochain/launcher/releases/) and play against a friend. (Launcher is still pre-alpha, so you may encounter some bugs.)
```
Comment (Connoropolous) [inline src/bluesky-try.md:25]: if they are on linux, there's a good likelihood they're CLI friendly, so omit the comment. If they are on mac, no cli needed

Comment (pdaoust) [reply src/bluesky-try.md:25]: Got it -- makes total sense.

**Review by pdaoust — APPROVED** [review]
LGTM; thanks for adding!

**Comment by pdaoust** [comment]
Deployed in PR #302

**Pull request merged**